docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -263,6 +263,45 @@ Most of the methods and constants in this library can be used or tested only in
 
 **To test the content of an update in Expo Go**, run [`eas update`](/eas-update/getting-started/) and then browse to the update in Expo Go. Note that this only simulates what an update will look like in your app, and most of the [Updates API](#api) is unavailable when running in Expo Go. Also note that only updates using [Expo Go-compatible libraries](/workflow/using-libraries/#determining-third-party-library-compatibility) are supported.
 
+## How to trace an update ID back to the Expo Dashboard
+
+When working with [EAS Updates](/eas-update/introduction/), you might encounter a scenario where you need to trace an `updateId` back to the [Expo dashboard](https://expo.dev/accounts/[account]). This can be challenging because `Updates.updateId` always returns an ID, regardless of whether [`Updates.isEmbeddedLaunch`](/#updatesisembeddedlaunch) is `true` or `false`. However, if the app is running an embedded update, attempting to look up the `updateId` in the [Expo dashboard](https://expo.dev/accounts/[account]) will result in an error. This happens because embedded updates are not tracked in the dashboard.
+
+### Determine if the Update is Embedded or Downloaded
+
+To avoid this issue, you can use the [`Updates.isEmbeddedLaunch`](/#updatesisembeddedlaunch) property to determine if the app is running an embedded update or one downloaded from the server. If `isEmbeddedLaunch` is `true`, the currently running update is embedded in the build, meaning it won't be available in the [Expo dashboard](https://expo.dev/accounts/[account]).
+
+Here's an example of how you can display whether the update is embedded or downloaded:
+
+```tsx UpdateStatus.tsx
+import * as Updates from 'expo-updates';
+import { ThemedText } from './components/ThemedText';
```

Comment:
```suggestion
import { Text } from 'react-native';
```

Let's use `Text` component from React Native which is primitive since we are not sure if developer will have ThemedText component in their app or not (from default template).